docs/nonlocal-networks.rst

```diff
@@ -0,0 +1,161 @@
+You can interact with any network defined in ``brownie-config.json`` programatically using the ``brownie.network.main`` methods.
+
+Connecting to a network:
+
+.. code-block:: python
+
+    >>> network.connect('ropsten')    
+    >>> network.is_connected()
+    True
+    >>> network.show_active()
+    'ropsten'
+
+Disconnecting from a network:
+
+.. code-block:: python
+
+    >>> network.disconnect()
+    >>> network.is_connected()
+    False
+
+Interacting with non-local networks
+==============================
+
+``brownie.module.rpc``
+--------
+
+The :ref:`rpc` module is unavailable when working with non-local networks.
+
+Accounts
+--------
+
+If you are connected to your own private node, Brownie will automatically load any unlocked accounts returned by your node.  In this case, there is no need to use ``accounts.load``.
+
+When interacting with a non-local network via a hosted node like Infura, you must provide the private key when loading your acccount in order to be able to sign transactions or deploy contracts
+
+.. code-block:: python
+
+        >>> accounts.add('8fa2fdfb89003176a16b707fc860d0881da0d1d8248af210df12d37860996fb2')
+        <Account object '0xc1826925377b4103cC92DeeCDF6F96A03142F37a'>
+        >>> accounts[0].balance()
+        17722750299000000000
+
+Once an account is added to the account object, the ``accounts.save`` and ``accounts.load`` can be used to save the accounts to an encrypted keystore and then load for later use.
+
+Transactions
+------------
+
+* Transaction status
+When submitting transactions on non-local networks, blocks are not immediately so transactions will likewise not be immediately confirmed. 
+Brownie does not provide a transaction receipt by default but will wait until the transaction has been confirmed before continuing execution.  
+Press ``Ctrl-C`` and a :ref:`api-network-tx` object will be returned with the pending transaction hash and can be stored to unique variables. ``TransactionReceipt.status`` will be ``-1`` until the transaction is mined and either succeeds or reverts.  
+
+* Debugging 
+The Brownie :ref:`debug` tools rely upon `debug_traceTransaction <https://github.com/ethereum/go-ethereum/wiki/Management-APIs#user-content-debug_tracetransaction>`__ RPC method which is not supported by `Infura <https://infura.io>`__. Attempts to call it will result in a ``RPCRequestError``.
+This means that the below ``TransactionReceipt`` attributes and methods are unavailable:
+
+* ``TransactionReceipt.return_value``
+* ``TransactionReceipt.trace``
+* ``TransactionReceipt.call_trace``
+* ``TransactionReceipt.traceback``
+* ``TransactionReceipt.source``
+
+:ref:`api-network-contract`
+---------
+Contracts
+*********
+
+On non-local networks, use the ``Contract`` class to interact with already deployed contracts.  
+
+
+You can instantiate the contract using ``contract.Contract`` method.  You will need to provide an ABI (typically as a JSON file) that is generated from the compiled contract code.  
```

Comment:
`abi` is given as a `dict` - while true that it's typically generated from a JSON file, stating that it expectas a JSON could result in someone trying to hand it a file object.

Reply:
Understood, I rephrased this to explain it needs to be a JSON interface object.